opencti-platform/opencti-graphql/src/domain/stixSightingRelationship.js

```diff
@@ -45,6 +46,9 @@ export const findById = (context, user, stixSightingRelationshipId) => {
   return storeLoadById(context, user, stixSightingRelationshipId, STIX_SIGHTING_RELATIONSHIP);
 };
 
+export function lazyLoadSpecVersion({ spec_version }) {
```

Comment:
This is not a lazyLoad function, a defaultValue one maybe.
And this is definitely not how we define function in our code. 

Finally I'm not sure there is any gain from defining that rather than just inlining it the way you used to do it

Reply:
> This is not a lazyLoad function, a defaultValue one maybe.

I agree with you because it better describes the function internal behavior. However *lazyLoad* is the term proposed and used all along in our discussion **to describe the behavior in the system** (i.e. get rid off `spec_version` in stored relations objects and only set it for graphql responses because some our clients need it, starting with `pycti` and probably externals too), hence the *lazy* in the code. 

Do you consider this term misleading? Totally fine to change the name if it makes more sense to you.

>  And this is definitely not how we define function in our code.

It will be changed.

> Finally I'm not sure there is any gain from defining that rather than just inlining it the way you used to do it

I agree it makes little sense for a oneline method, however, I consider it a good practice as it adds lisiblity. With the function definition, I can focus on the *what* is done instead of *how* it is done. The intent is explicit and I can have a better high-level understanding of what is going on, instead of having to make sense of all the details.

Reply:
As discussed today : It seems that spec_version is missing from StixRelationship type. We need to check with @richard-julien.

In that case we need to update only the resolver/domain of stixRelationship and rename the method.